overlord/patch/patch5.go

```diff
@@ -71,7 +71,7 @@ func patch5(st *state.State) error {
 			return err
 		}
 
-		err = wrappers.AddSnapServices(info, nil, nil, log)
+		err = wrappers.AddSnapServices(info, nil, log)
```

Comment:
do we need to more here, given that this doesn't enable anymore?

Reply:
We call wrappers.StartServices below which enables all services.

Reply:
patch5 lacks tests, I can add them perhaps as a followup.